app/controllers/monologue/application_controller.rb

```diff
@@ -10,7 +10,7 @@ def recent_posts
   end
 
   def all_tags
-    @tags  =  Monologue::Tag.with_frequency.order("name")
+    @tags = Monologue::Tag.order("name").select{|t| t.frequency>0}
```

Comment:
Surrounding space missing for operator '>'.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.